hpx/runtime/parcelset/decode_parcels.hpp

```diff
@@ -186,6 +185,21 @@ namespace hpx { namespace parcelset
         }
     }
 
+    ///////////////////////////////////////////////////////////////////////////
+    template <typename Parcelport, typename Buffer>
+    void decode_message(
+        Parcelport & pp
+      , Buffer buffer
+      , std::size_t parcel_count
+      , std::size_t num_thread = -1
+    )
+    {
+        std::vector<serialization::serialization_chunk>
+            chunks(std::move(decode_chunks(buffer)));
```

Comment:
Is this `std::move()` really necessary? decode_chunks already returns an rvalue, this should trigger move-initialization without explicitly moving the result.